build/CMakeLists.txt

```diff
@@ -6,8 +6,8 @@ set(PROJECT_NAME hookman)
 set(ARTIFACTS_DIR ${CMAKE_CURRENT_SOURCE_DIR}/artifacts)
 
 if(NOT WIN32)
-  set(CMAKE_C_COMPILER    clang)
-  set(CMAKE_CXX_COMPILER  clang++)
+  set(CMAKE_C_COMPILER    cc)
```

Comment:
What happens if these 4 lines are removed, and we leave the defaults? (If the result is the same, it's better to remove the lines)

Reply:
I can try it. You mean, remove the whole `if`?

Reply:
Yes